.github/actions/release-automator/action.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Release Automator'
+author: GreatG0ose
+description: 'Sends sign-off requests, pre-/post-release emails, generates tweet'
+inputs:
+  command:
+    description: 'Command to execute'
+    required: true
+  version:
+    description: 'Release version'
+    required: true
+runs:
+  using: 'docker'
+  image: 'ghcr.io/greatg0ose/release-automator:sha-94d43d2'
```

Comment:
Is this using an image from private repo? Maybe this should be moved to an unzer repo?

Reply:
Sry, I did not mean private in that way. But it it is on  the personal repository instead of the unzer one.

Reply:
Should be fine for now. But i would rather move/fork it to a unzer repo.